DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -16,19 +16,19 @@ @implementation _DTXTimerTrampoline
 {
 	id _target;
 	SEL _sel;
-	
```

Comment:
Sorry for the whitespace noise - my editor keeps wanting to remove them.
This is in a separate commit.

Reply:
Don't worry about that.